content/en/blog/2024/otel-generative-ai/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: OpenTelemetry for Generative AI
+linkTitle: OpenTelemetry for Generative AI
+date: 2024-11-09
+author: >-
+  [Drew Robbins](https://github.com/drewby) (Microsoft),  [Liudmila
+  Molkova](https://github.com/lmolkova) (Microsoft)
+---
+
+As organizations increasingly adopt Large Language Models (LLMs) and other
+generative AI technologies, ensuring reliable performance, efficiency, and
+safety is essential to meet user expectations, optimize resource costs, and
+safeguard against unintended outputs. Effective observability for AI operations,
+behaviors, and outcomes can help meet these goals. OpenTelemetry is being
+enhanced to support these needs specifically for generative AI.
+
+Two primary assets are in development to make this possible: **Semantic
+Conventions** and an **Instrumentation Library**.
+
+**Semantic Conventions** establish standardized guidelines for how telemetry
```

Comment:
```suggestion
[**Semantic Conventions** ](https://opentelemetry.io/docs/concepts/semantic-conventions/) establish standardized guidelines for how telemetry
```

I'm fine with the inline explanation (and maybe we can even take some if and improve the existing concept page), but link to docs pages where possible, so people can dig deeper if they want to learn more